tests/e2e/fixtures/cognito_dependencies.py

```diff
@@ -118,12 +123,14 @@ def callback():
 
 @pytest.fixture(scope="class")
 def post_deployment_dns_update(
-    metadata, region, request, cluster, cognito_bootstrap, kustomize
+    metadata, region, request, cluster, cognito_bootstrap, installation
 ):
-
+    print ("get into post deployment...")
```

Comment:
nit: remove

Reply:
removed it. put there for debugging purpose